controllers/ControllerDecorator.ts

```diff
@@ -17,11 +17,19 @@ import {ErrorHandlerManager, ERRORHANDLER_KEY, DEFAULT_ERROR_HANDLER} from '../e
 import httpStatus = require('http-status');
 
 let controllers: ControllerRegistration[] = [],
-    definedRoutes = [];
+    definedRoutes = [],
+    bodyParserInstalled = false;
 
 const PRIMITIVE_TYPES = [Object, String, Array, Number, Boolean],
     NON_JSON_TYPES = [String, Number, Boolean];
 
+try {
+    require('body-parser');
```

Comment:
Sadly it's not possible to check whether `body-parser` is only installed or actually used. So that's the "most possible check" we can do.